### DISCUSSION TITLE
v2.1.1 fails to load on Windows

### DISCUSSION THREAD

**Discussion by Dr-Flay** [DNSCrypt/dnscrypt-proxy#1910] [Potential issues]
The recent build fails to enable the service on either of my Win 7 or Win 10 systems (x64).
2.1.0 works fine.
No useful errors in Windows event viewer so I can't give much more detail other than it "fails unexpectedly".

**Comment by dodmi** [comment]
It works fine on my system.
dnscryptproxy doesn't log to windows event log.
Have a look in your config for `log_file` to get the location (or set it). This log will help you to track down the problem.

**Comment by Dr-Flay** [comment]
I switched on the logging in the advanced settings in Simple DNSCrypt.
The log shows this error;

[2021-10-06 23:07:41] [NOTICE] dnscrypt-proxy 2.1.1
[2021-10-06 23:07:41] [FATAL] Unsupported key in configuration file: [daemonize]

As I didn't make the config I can only assume the GUI is now making an incompatible config for this build.
Key removed and now it loads.

**Reply by dodmi** [reply]
:-)
